packages/client3/components/design-system/input/select.tsx

```diff
@@ -187,21 +187,22 @@ export function Select({
       </Listbox>
 
       {helperText || errorMessage ? (
-        <HelperText
-          className={clsx(
-            isError
-              ? "text-clay-500"
-              : colorScheme === "light"
-              ? "text-sand-500"
-              : colorScheme === "dark"
-              ? "text-sand-300"
-              : null,
-            "mt-1 text-sm leading-none"
-          )}
-        >
-          {errorMessage ? errorMessage : helperText}
-        </HelperText>
-      ) : null}
+        <></>
+      ) : // <HelperText
+      //   className={clsx(
+      //     isError
+      //       ? "text-clay-500"
+      //       : colorScheme === "light"
+      //       ? "text-sand-500"
+      //       : colorScheme === "dark"
+      //       ? "text-sand-300"
+      //       : null,
+      //     "mt-1 text-sm leading-none"
+      //   )}
+      // >
+      //   {errorMessage ? errorMessage : helperText}
+      // </HelperText>
```

Comment:
Old component used by goldfinch. I commented instead of deleting for future references.